testsuite/smoke-test/src/operational_tooling.rs

```diff
@@ -22,7 +24,51 @@ use libra_types::{
     chain_id::ChainId,
     transaction::authenticator::AuthenticationKey,
 };
-use std::convert::{TryFrom, TryInto};
+use std::{
+    convert::{TryFrom, TryInto},
+    fs,
+    str::FromStr,
+};
+
+#[test]
+fn test_account_resource() {
+    let mut swarm = SmokeTestEnvironment::new(1);
+    swarm.validator_swarm.launch();
+
+    // Load a node config
+    let node_config =
+        NodeConfig::load(swarm.validator_swarm.config.config_files.first().unwrap()).unwrap();
+
+    // Connect the operator tool to the first node's JSON RPC API
+    let op_tool = swarm.get_op_tool(0);
+
+    // Load validator's on disk storage
+    let backend = load_backend_storage(&node_config);
```

Comment:
can we make this a function instead? just to get a backend or a storage for a given validator node?

Reply:
Yeah, I actually have a bigger refactor that I'm planning to do which will take care of all these annoying repeated patterns. So I'm gonna land this for now, and then in the next PR use the refactor 😄